MAINTAINERS

```diff
@@ -0,0 +1,16 @@
+Avocado-vt maintainers
+======================
+
+The intention of this file is not to establish who owns what portions of the
+code base, but to provide a set of names that developers can consult when they
+have a question about a particular subset and also to provide a set of names
+you can look for on github for any pull requests you might want to get approved.
+
+In general, if you have a question, you should send an email to the Virt Test
+development mailing list and not any specific individual privately.
+
+L: Avocado-devel <avocado-devel@redhat.com>
+M: Yanbing Du <ydu@redhat.com>
+M: Jiangang Wei <weijg.fnst@cn.fujitsu.com>
+M: Guannan Wayne Sun <gsun@redhat.com>
+M: Hao Liu <hliu@redhat.com>
```

Comment:
IMO @xutian that would make the list too big. The purpose of this file is not to list all active members, but the most active people you should ping if you have some problem.

@clebergnu and me ( @ldoktor ) are still active, but they are not the primary contacts. So let's reconsider the remaining people, these were listed in the email thread (RH):

M: Yanbing Du <ydu@redhat.com>
M: Guannan Wayne Sun <gsun@redhat.com>
M: Hao Liu <hliu@redhat.com>

Do you want to to remove some of them and place someone else instead? (or reorder, I used `git shortlog -s -n --since=2015-04-19` and sorted people accordingly)